src/parameterizations/lateral/MOM_MEKE.F90

```diff
@@ -175,8 +175,7 @@ subroutine step_forward_MEKE(MEKE, h, SN_u, SN_v, visc, dt, G, GV, US, CS, hu, h
   if (.not.associated(MEKE)) call MOM_error(FATAL, &
          "MOM_MEKE: MEKE must be initialized before it is used.")
 
-  if ((CS%MEKE_damping > 0.0) .or. (CS%MEKE_Cd_scale > 0.0) .or. (CS%MEKE_Cb>0.) &
```

Comment:
I do not think that the logic change here is correct.   This change will cause lines 517-532 not to be executed, even if CS%MEKE_damping is positive, which I think is wrong.

Reply:
Thanks, @Hallberg-NOAA. 
@sdbachman and I have modified the code to address your comment.  Now, drag_rate is always initialized, either to zero or to the bottom drag formula. We also updated the logic to always execute the second Strang splitting calculation. We noticed that drag_rate_J15 is never used, so we deleted this variable. We also deleted some commented code.

Reply:
It seems the last commit is changing answers. I will look into this and report back here.

Reply:
I accidentally deleted some code which led to an answer change (thankfully CI tests caught that). After re-introducing the code, all tests are passing.

Reply:
With the recent additional changes, I am satisfied that this issue has been addressed.